### PR TITLE
fix(ui): display if sale is complete in stostatus component

### PR DESCRIPTION
**Pull request by sajclarke**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [x] I've added this PR's link to its Asana task(s)
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

File: packages/polymath-ui/src/components/STOStatus/STOStatus.js [hunk 1]
```diff
@@ -133,23 +137,47 @@ export default class STOStatus extends Component<Props> {
...
+              >
+                <p align="center">
+                  <Icon
+                    name="checkmark--outline"
+                    fill="#00AA5E"
+                    width="64"
+                    height="64"
+                  />
+                </p>
+                <h2 className="pui-h2" align="center">
```
Comment (Grsmto) [inline packages/polymath-ui/src/components/STOStatus/STOStatus.js]: You should use the new components for this.
`<Heading variant="h2" textAlign="center" />`
https://polymath-ui.netlify.com/src-components-heading-heading

File: packages/polymath-ui/src/components/STOStatus/STOStatus.js [hunk 2]
```diff
@@ -133,23 +137,47 @@ export default class STOStatus extends Component<Props> {
...
-                pausable={!notPausable}
-              />
+              <ContentBox
+                style={{
+                  borderRadius: '10px',
+                  borderTop: '15px solid #00AA5E',
+                  width: '350px',
+                }}
+              >
+                <p align="center">
```
Comment (Grsmto) [inline packages/polymath-ui/src/components/STOStatus/STOStatus.js]: Same here, use new component. `align` HTML property is invalid HTML.
`<Paragraph textAlign="center">`

File: packages/polymath-ui/src/components/STOStatus/STOStatus.js [hunk 3]
```diff
@@ -133,23 +137,47 @@ export default class STOStatus extends Component<Props> {
...
-                isPaused={this.props.isStoPaused}
-                pausable={!notPausable}
-              />
+              <ContentBox
+                style={{
+                  borderRadius: '10px',
+                  borderTop: '15px solid #00AA5E',
+                  width: '350px',
+                }}
+              >
```
Comment (Grsmto) [inline packages/polymath-ui/src/components/STOStatus/STOStatus.js]: Just FYI on the UI v2 we have a new component for this in `packages/new-polymath-ui/src/components/CardFeatureState`.
So it's fine to leave it like this for legacy.

**Comment by RafaelVidaurre** [comment]
@Grsmto is this good to merge now that @sajclarke implemented your requested changes?

**Comment by Grsmto** [comment]
@RafaelVidaurre yep all good now.

**Pull request merged**